### DISCUSSION TITLE
230710 페르소나 재정의 및 검토 (17:00 ~ 18:00)

### DISCUSSION THREAD

**Discussion by seokhwan-an** [woowacourse-teams/2023-shook#34] [회의]
### 회의 안건
페르소나를 재정의 하고 검토하는 시간을 가진다.

**Comment by Cyma-s** [comment]
**킬링파트를 들으러 오는 유저는 왜 킬링파트를 들으려 했을까?**

→ 새로운 노래를 찾으려고, 그냥 가벼운 마음으로, 사람들이 어떤 부분을 킬링파트로 지정했을지 궁금해서

**킬링파트를 등록하려는 유저는 왜 킬링파트를 등록하려 했을까? (팬심)**

제일 큰 소비자 층 → 특정 가수의 팬들

→ 노래 / 가수에 대한 팬심

→ 서비스를 활성화하는 도구
